tests/resources/integration_test_data/assetized_child_ref_non_assetized_grandchild/bal_library.json

```diff
@@ -0,0 +1,15 @@
+{
+  "entities": {
+    "floor": {
+      "versions": [
+        {
+          "traits": {
+            "openassetio-mediacreation:content.LocatableContent": {
+              "location": "floor1.usd"
```

Comment:
[fix] These need to be `file:///` URLs

Reply:
[a38c6af](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/a38c6af2977c4e2ba0a3b0c98fb37539bbad6286)